test/unit/units/ssh_access_test.py

```diff
@@ -18,23 +18,70 @@ def test_migration_continues_on_error(
         main()  # expect pass and comment
         assert mock_error.called
 
+    @patch('suse_migration_services.command.Command.run')
+    @patch('re.findall')
+    @patch('shutil.copy')
+    @patch('os.path.exists')
     @patch.object(Defaults, 'get_ssh_keys_paths')
     @patch('glob.glob')
-    @patch.object(Defaults, 'get_migration_ssh_file')
     @patch('suse_migration_services.logger.log.info')
     def test_main(
-        self, mock_info, mock_get_migration_ssh_file,
-        mock_glob_glob, mock_glob_ssh
+        self, mock_info, mock_glob_glob, mock_get_ssh_key_path,
+        mock_path_exists, mock_shutil_copy, mock_re_findall,
+        mock_Command_run
     ):
-        mock_glob_ssh.return_value = ['/path/']
-        mock_get_migration_ssh_file.return_value = '../data/authorized_keys'
-        mock_glob_glob.return_value = [
-            '../data/authorized_azure_keys',
-            '../data/authorized_ec2_keys'
+        mock_get_ssh_key_path.return_value = ['/path/']
+        mock_glob_glob.side_effect = [
+            [
+                '/path/authorized_azure_keys',
+                '/path/authorized_ec2_keys'
+            ],
+            [
+                '/system-root/etc/ssh/ssh_host_ecdsa_key.pub',
+                '/system-root/etc/ssh/ssh_host_ecdsa_key'
+            ]
```

Comment:
I still don't get why the test is messing with the authorized_keys testing. For the purpose of adding the host keys there should not be any change needed on the side of the auth key setup. Please explain the motivation for this change and if we need it put it in an extra pull request. Thanks

Reply:
github quoted this a bit unfortunate. The lines 30-37 are meant by this comment

Reply:
There is no change in the authorized_keys testing in those lines, the glob returns the values for authorized_key (as before) and the values for host keys (new values for the new case)

Reply:
I reverted all the changes related to authorized_keys

Reply:
Thanks